drivers/sensors/pmw33xx_common.h

```diff
@@ -79,6 +81,19 @@ _Static_assert(sizeof((pmw33xx_report_t){0}.motion) == 1, "pmw33xx_report_t.moti
 #    endif
 #endif
 
+// Support single spelling and default to be the same as left side
+#if !defined(PMW33XX_CS_PINS_RIGHT)
+#    if !defined(PMW33XX_CS_PIN_RIGHT)
+#        define PMW33XX_CS_PIN_RIGHT PMW33XX_CS_PIN
+#    endif
+#    define PMW33XX_CS_PINS_RIGHT { PMW33XX_CS_PIN_RIGHT }
```

Comment:
lint has some issues.  should be able to use `qmk format-c` on this file to fix them.